flatpak/manifest_tool/sources_ops.py

```
@@ -1,523 +1,22 @@
-        return False
-    present = any(
-        isinstance(source, dict)
-        and source.get("type") == "file"
-        and source.get("path") == path
-        for source in sources
-    )
-    if present:
-        return False
-    sources.append({"type": "file", "path": path})
-    return True
+from .operations.sources import *
```

Ruff (0.13.1)

`from .operations.sources import *` used; unable to detect undefined names

(F403)